common/s2n-codec/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-codec"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
dont need to bump this if nothing changed here.

Reply:
This crate was updated in https://github.com/aws/s2n-quic/pull/1549